lib/model/folderstate.go

```diff
@@ -52,6 +52,14 @@ func (s folderState) String() string {
 	}
 }
 
+type remoteFolderState int
+
+const (
+	remoteMissing remoteFolderState = iota
```

Comment:
Maybe `remoteNotSharing`? Without context I doubt it's clear what it means for a folder to be "missing".

Reply:
Done.